cpp/ps0003/pooo3-soln.cpp

```diff
@@ -1,17 +1,23 @@
-# include <iostream>
+#include <iostream>
+
 using namespace std;
+
 int main(){
-int number;
-cout<<"Enter your number:";
-cin>> number;
-if (number%2==0)
-{
-cout<<"the number "<<number<<" is even";
+  // prompting user for number 2/2
+  int number;
+  cout << "Enter your number:";
+  cin >> number;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number